components/storage/blockdevice/COMPONENT_QSPIF/mbed_lib.json

```diff
@@ -0,0 +1,20 @@
+{
+"name": "qspif",
+    "config": {
+        "QSPI_FREQ": "40000000"
+    },
+    "target_overrides": {
+        "DISCO_F413ZH": {
+             "QSPI_FREQ": "80000000"
+        },
+       "DISCO_L475VG": {
```

Comment:
Should be DISCO_L475VG_IOT01A ?

Reply:
Also, config pin options from external driver are not present here. https://github.com/ARMmbed/qspif-blockdevice/blob/master/mbed_lib.json

Reply:
@jeromecoutant  - 10x I'll update
@deepikabhavnani - the Pins were there at the beginming, and I was asked to remove them because it was decided to rely on lower level h files to standardize pin names:
    QSPI_FLASH1_IO0
    QSPI_FLASH1_IO1
    QSPI_FLASH1_IO2 
    QSPI_FLASH1_IO3 
    QSPI_FLASH1_SCK
    QSPI_FLASH1_CSN